### DISCUSSION TITLE
Zoom in a specific slot/group and rotate individually

### DISCUSSION THREAD

**Discussion by osnynetto** [google/model-viewer#4962] [Q&A]
Let's consider this 3D model with multiple objects in the scene. Each object is grouped so it can be targeted individually. My goal is to allow the user to click on a specific object (e.g., a chair) to zoom in on it—this part is already working using [this example](https://modelviewer.dev/examples/annotations/index.html#cameraViews). Once zoomed in, I want to make only that object's group (or "slot") rotatable while locking the rotation for other groups. The sibling groups would remain locked until the user zooms out to view all the objects together.

How feasible is this?

![image](https://github.com/user-attachments/assets/06e8d969-098c-4752-a9f1-ee95b04102a2)

**Comment by elalish** [comment]
We don't currently expose an API at that level. #4904 and #4940 are moving in that direction.
